css/style.css

```diff
@@ -45,25 +45,155 @@ Developer @GA in NYC
 /* BASE STYLES
 /*******************************************/
 
+#container {
+  border: 5px;
+  border-color: #606161;
+  width: 100%;
+  margin: auto;
+  height: 500px;
+  padding-left: 70px;
+  padding-right: 70px;
+}
+
+
 body {
-     color: #000;
+  font-family: 'Open Sans';
+}
+
+header {
+  background: url(../images/header_bg.jpg) no-repeat center top rgb(134,141,157);
+  color: white;
+  padding-bottom: 200px;
+  padding-left: 50px;
+  margin: auto;
+  text-align: center;
+
+}
+
+header h2 {
+  text-align: center;
+  padding: 50px 100px;
+}
+
+h1 {
+  font-size: 24px;
+  float: left;
+  padding-bottom: 300px;
+  margin-right: 50px;
+}
+
+section {
+     background-color: #f0efef;
+     color: black;
      font-size: 12px;
      line-height: 1.4;
-     font-family: Helvetica, Arial, sans-serif;
-     background: url(../images/header_bg.jpg) no-repeat center top rgb(134,141,157);
-      background-size: cover;
+     background-size: cover;
+    padding: 40px; 10px;
+    border: 1px #f0efef;
 }
 
+button {
+    background-color: #f9e42e;
+    border-color: #f9e42e;
+    padding: 10px 25px;
+    text-align: center;
+    font-size: 14px;
+    font-weight: bolder;
+    border-radius: 5px;
+    color: #033048;
+}
+
+h3 {
+  color: #033048;
+  font-size: 20px;
+
+}
+
+h3 li {
+  font-size: 13px;
+  font-weight: lighter;
+}
+
+
+h3 {
+  color: #033048;
+  font-size: 24px;
+  font-weight: semibold;
+}
+
+h3 p {
```

Comment:
same goes here. you can't nest `<p>` inside of `<h3>` (or any `<h#>` for that matter)